src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Ball.Color.values().length);
```

Comment:
This code creates a new array of `Ball.Color.values()` each time it is called, which is not efficient. You can store the result of `Ball.Color.values()` in a private static final array and reuse it.